bridge/hardhat.config.js

```diff
@@ -57,10 +54,15 @@ module.exports = {
     ],
   },
   namedAccounts: getNamedAccounts(),
+  // etherscan: {
+  //   apiKey: "anyapi"
+  // },
```

Comment:
I'd uncomment this and maybe change `anyapi` for `YOUR_ETHERSCAN_API_KEY_HERE`

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/4068c20b90d789cd4164cdffe699095cb0a665a4